coverage_comment/coverage.py

```diff
@@ -255,8 +284,18 @@ def get_diff_coverage_info(
         total_num_lines += count_total
         total_num_violations += count_missing
 
+        num_branches_covered = None
+        num_branches_total = None
+        if coverage.meta.branch_coverage:
+            num_branches_covered = 12
+            num_branches_total = 12
```

Comment:
lol 12?

Reply:
(It's my code, so just to make it clear, I'm lmao to my own shortcomings. Your contribution is very good in comparison :D )